lib/overcommit/hook/pre_push/r_spec.rb

```diff
@@ -1,14 +1,47 @@
 module Overcommit::Hook::PrePush
   # Runs `rspec` test suite before push
+  # .overcommit.yml configuration:
+  # Set `changed: true` to run rspec only over edited files.
+  # Set `remote`: $remote_name` to specify a remote name.
   #
   # @see http://rspec.info/
   class RSpec < Base
     def run
-      result = execute(command)
+      command = ['rspec']
```

Comment:
You don't want to explicitly override `command` here with a local variable, as this breaks your ability to configure additional flags via the `flags` hook option.